crates/op-rbuilder/src/args/op.rs

```diff
@@ -71,12 +74,54 @@ pub struct OpRbuilderArgs {
         env = "PLAYGROUND_DIR",
     )]
     pub playground: Option<PathBuf>,
+    /// List or builders in the network that FCU would be propagated to
+    #[arg(long = "builder.engine-api-peer", value_parser = parse_engine_peer_arg, action = clap::ArgAction::Append)]
+    pub engine_peers: Vec<EnginePeer>,
 }
 
-fn expand_path(s: &str) -> Result<PathBuf, String> {
+fn expand_path(s: &str) -> Result<PathBuf> {
     shellexpand::full(s)
-        .map_err(|e| format!("expansion error for `{s}`: {e}"))?
+        .map_err(|e| anyhow!("expansion error for `{s}`: {e}"))?
         .into_owned()
         .parse()
-        .map_err(|e| format!("invalid path after expansion: {e}"))
+        .map_err(|e| anyhow!("invalid path after expansion: {e}"))
+}
+
+/// Parse engine peer configuration string for clap argument parsing.
+///
+/// Format: "url@jwt_path" (JWT path is required)
+/// - url: HTTP/HTTPS endpoint of the peer builder
+/// - jwt_path: File path to JWT token for authentication (required after @)
+fn parse_engine_peer_arg(s: &str) -> Result<EnginePeer> {
+    let s = s.trim();
+
+    if s.is_empty() {
+        return Err(anyhow!("Engine peer cannot be empty"));
+    }
+
+    // Find the @ delimiter - it's required
+    let Some(at_pos) = s.find('@') else {
+        return Err(anyhow!(
+            "Engine peer must include JWT path after '@' (format: url@jwt_path)"
+        ));
+    };
+
+    let url_part = &s[..at_pos];
+    let jwt_path_part = &s[at_pos + 1..];
```

Comment:
nit: this will crash the system with an unhelpful error message for this string: `url@`.  Le't make sure that `@` is not the last character.